File: Codes/2255-count-prefixes-of-a-given-string.py
'''
Author: LetMeFly
Date: 2025-03-24 17:52:11
LastEditors: LetMeFly.xyz
LastEditTime: 2025-03-24 17:52:19
'''
from typing import List

class Solution:
    def countPrefixes(self, words: List[str], s: str) -> int:
        return sum(s.startswith(word) for word in words)

File: Codes/2255-count-prefixes-of-a-given-string.java
/*
 * @Author: LetMeFly
 * @Date: 2025-03-24 17:53:35
 * @LastEditors: LetMeFly.xyz
 * @LastEditTime: 2025-03-24 17:53:41
 */
class Solution {
    public int countPrefixes(String[] words, String s) {
        int ans = 0;
        for (String word : words) {
            ans += s.startsWith(word) ? 1 : 0;
        }
        return ans;
    }
}